src/circllhist.h

```diff
@@ -64,6 +64,23 @@ API_EXPORT(double) hist_bucket_to_double_bin_width(hist_bucket_t hb);
 API_EXPORT(hist_bucket_t) double_to_hist_bucket(double d);
 //! Create the bucket that value * 10^(scale) belongs to
 API_EXPORT(hist_bucket_t) int_scale_to_hist_bucket(int64_t value, int scale);
+//! Writes a standardized string representation to buf
+//! Buf must be of size 9 (8chars+terminator) or larger.
+//!
+//! Format spec: "sxxetyyy", where
+//! - s = '+' or '-' global sign
+//! - xx -- two digits representing val as decimal integer (in 10 .. 99)
+//! - e = 'e' literal character
+//! - t = '+' or '-' exponent sign
+//! - yyy -- three digits representing exp as decimal integer with leading 0s
+//!
+//! Exception: The zero bucket is represented  as "0"
+//! Exception: Invalid buckets are represented as "NAN"
+//!
+//! Examples:
+//!      1  => '+10e-001'; 12    => '+12e+000';
+//!  -0.23  => '-23e-003'; 23000 => '+23e+003';
+API_EXPORT(void) hist_bucket_to_string(hist_bucket_t hb, char *buf);
```

Comment:
I can do that, but note that this constant appears no-where inside the code.
The buffer is filed by explicitly referencing bytes. No iteration occurs.